composer.json

```
@@ -31,7 +31,7 @@
         "doctrine/inflector": "^1.4 || ^2.0",
         "doctrine/instantiator": "^1.3 || ^2",
         "doctrine/lexer": "^3",
-        "doctrine/persistence": "^3.3.1 || ^4",
+        "doctrine/persistence": "^3.3.1 || ^4@dev",
```

To commit after the release of `doctrine/persistence: 4.1.0`
```suggestion
        "doctrine/persistence": "^3.3.1 || ^4",
```

---

It has been released 👍

---

I see it released, but I don't find it possible to install it. 
Do you know if there are any issues with Packagist?

```
app@79c38110bfdf:/app$ composer req doctrine/persistence ^4.1
./composer.json has been updated
Running composer update doctrine/persistence
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires doctrine/persistence ^4.1, found doctrine/persistence[4.1.x-dev, 4.2.x-dev] but it does not match your minimum-stability.
```

It looks like that composer doesn't get the package version:

```
app@79c38110bfdf:/app$ composer show doctrine/persistence --all
name     : doctrine/persistence
descrip. : The Doctrine Persistence project is a set of shared interfaces and functionality that the different Doctrine object mappers share.
keywords : orm, persistence, object, odm, mapper
versions : 5.0.x-dev, 4.2.x-dev, 4.1.x-dev, 4.0.x-dev, * 4.0.1, 4.0.0, 3.4.x-dev, 3.4.0, 3.3.x-dev, 3.3.3, 3.3.2, 3.3.1, 3.3.0, 3.2.x-dev, 3.2.0, 3.1.x-dev, 3.1.4, 3.1.3, 3.1.2, 3.1.1, 3.1.0, 3.0.x-dev, 3.0.4, 3.0.3, 3.0.2, 3.0.1, 3.0.0, 2.5.x-dev, 2.5.7, 2.5.6, 2.5.5, 2.5.4, 2.5.3, 2.5.2, 2.5.1, 2.5.0, 2.4.x-dev, 2.4.2, 2.4.1, 2.4.0, 2.3.x-dev, 2.3.0, 2.2.x-dev, 2.2.4, 2.2.3, 2.2.2, 2.2.1, 2.2.0, 2.1.x-dev, 2.1.1, 2.1.0, 2.0.x-dev, 2.0.1, 2.0.0, 1.4.x-dev, 1.3.x-dev, 1.3.8, 1.3.7, 1.3.6, 1.3.5, 1.3.4, 1.3.3, 1.3.2, 1.3.1, 1.3.0, 1.2.x-dev, 1.2.0, 1.1.x-dev, 1.1.1, v1.1.0, 1.0.x-dev, v1.0.1, v1.0.0, dev-3.4.x-merge-up-into-4.0.x_rJVj7XvE, dev-4.1.x-merge-up-into-5.0.x_TA2qrOVB, dev-4.0.x-merge-up-into-4.1.x_c0gHyWAW, dev-3.3.x-merge-up-into-3.4.x_eHj8ljQl
released : 2025-08-01, 3 weeks ago
```

---

It doesn't seem to be returned from Packagist: https://repo.packagist.org/p/doctrine/persistence.json

---

For now, the issue has been resolved (see https://github.com/composer/packagist/issues/1569#issuecomment-3219327655).